forch/faucetizer.py

```diff
@@ -18,14 +18,32 @@
 
 class Faucetizer:
     """Collect Faucet information and generate ACLs"""
-    def __init__(self, structural_faucet_config, segments_to_vlans):
+    def __init__(self, orch_config, structural_faucet_config, segments_to_vlans,
+                 behavioral_config_file):
         self._dynamic_devices = {}
         self._static_devices = {}
         self._segments_to_vlans = segments_to_vlans
         self._structural_faucet_config = structural_faucet_config
         self._behavioral_faucet_config = None
-        self._lock = threading.Lock()
-
+        self._config = orch_config
+        self._behavioral_config_file = behavioral_config_file
+        self._lock = threading.RLock()
+
+    # pylint: disable=no-self-argument, protected-access, no-method-argument
+    def _write_behavioral_config():
```

Comment:
this is not a good use of annotations -- it's taking something that can/should be very simple and adding an unnecessary layer of abstraction on top of it -- makes it very hard to read/understand the code. This really should just be a regular function that's called as needed.

Reply:
Ok, removed annotations and created _write_behavioral_config to wrap the oeprations.